Fixed field names to maintain consistency with the convention used.

derikson-dev opened the pull request:
## What does this PR do?

Fixed field names to use capital letters for initial letters to keep name convention.

https://github.com/o3de/o3de/issues/18922

## How was this PR tested?

Checked the **Enter Project Details** (Project Manager > New Project > Create New Project) screen.

derikson-dev committed:
Fixed field names to maintain consistency with the convention used.

Signed-off-by: derikson-dev <85710361+derikson-dev@users.noreply.github.com>

mprettner-huawei commented on Code/Tools/ProjectManager/Resources/ProjectManager.aps:
This file should probably not be checked in to git

derikson-dev replied:
@mprettner-huawei I didn't get it, Can you elaborate more, please?

guillaume-haerinck replied:
Ouch I missed that, yeah seems like the .gitignore will have to be updated but that would be for another PR, for now you can just add a commit where you delete this file, end result of the PR will only contain your .cpp change

derikson-dev replied:
Done.

derikson-dev committed:
Remove obsolete file: ProjectManager.aps

derikson-dev committed:
DCO Remediation Commit for derikson-dev <85710361+derikson-dev@users.noreply.github.com>
I, derikson-dev 85710361+derikson-dev@users.noreply.github.com, hereby add my Signed-off-by to this commit: 5a9714bf5d266f9dec25ec607deb3f29112c1955

Signed-off-by: derikson-dev 85710361+derikson-dev@users.noreply.github.com

derikson-dev committed:
DCO Remediation Commit for derikson-dev <85710361+derikson-dev@users.noreply.github.com>
DCO Remediation Commit for derikson-dev <85710361+derikson-dev@users.noreply.github.com>

I, derikson-dev <85710361+derikson-dev@users.noreply.github.com>, hereby add my Signed-off-by to this commit: 2d6e7cf5b294d025e7d4c55275edd5a3c4d70021
I, derikson-dev <85710361+derikson-dev@users.noreply.github.com>, hereby add my Signed-off-by to this commit: 66357e8d387687bbb54e50ef66000768e9708136

Signed-off-by: derikson-dev <85710361+derikson-dev@users.noreply.github.com>
